package.json

```diff
@@ -32,9 +32,11 @@
     "@babel/runtime": "^7.12.13",
     "@date-io/luxon": "1.x",
     "@material-ui/core": "^4.11.3",
+    "@material-ui/data-grid": "^4.0.0-alpha.32",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.10",
+    "@material-ui/styles": "^4.11.4",
```

Comment:
Doesn't look like you are using this specifically. It should be imported by other Material UI packages.
```suggestion
```